dylint_lints/de02_api_layer/de0203_dtos_must_use_api_dto/README.md

```
@@ -0,0 +1,83 @@
+# DE0203: DTOs Must Use api_dto Macro
+
+### What it does
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)